gtkm/static/script.js

```diff
@@ -0,0 +1,46 @@
+let githubUserData = {
+  githubName: "Name",
+  githubSurname: "Surname",
+  githubUsername: "namesurname",
+};
+
+const getUserData = function (dataObject) {

```

Comment:
Jeśli mam być czepialski to, czemu przekazujesz ten obiekt do tej funkcji, skoro i tak jest on globalnie dostępny?

Poza tym nazwa funkcji jest myląca. Przedrostek `get` sugeruje, że to jest jakiś getter zwracający coś.